series/series_test.go

```diff
@@ -449,6 +534,36 @@ func TestSeries_Subset(t *testing.T) {
 			Bools([]bool{true, false, false, true, true}),
 			"[A K D]",
 		},
+		{
```

Comment:
If the PR is about adding support to `time.Time` try not to add things that have nothing to do with it. These tests are nice to have though, but please submit a separate PR next time.